### DISCUSSION TITLE
ffmpeg_recoverable_errors delimiter

### DISCUSSION THREAD

**Discussion by gniknalu** [roflcoopter/viseron#228] [Q&A] [answered]
Is the `ffmpeg_recoverable_errors` delimiter a comma or a semi-colon or something else?

**Comment by roflcoopter** [comment, marked as answer]
Its a list in yaml, so each entry starts with `-`
For example the default values are this:
```yaml
ffmpeg_recoverable_errors:
  - "error while decoding MB"
  - "Application provided invalid, non monotonically increasing dts to muxer in stream"
```

Does that answer your question?

**Reply by gniknalu** [reply]
Yes it does. Thank you again!
